src/SwedbankPay.Sdk/PaymentInstruments/Trustly/ITrustlyPaymentsResource.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Trustly
+{
+    public interface ITrustlyResource
+    {
+        /// <summary>
+        ///     Creates a new Trustly payment
+        /// </summary>
+        /// <param name="paymentRequest"></param>
+        /// <param name="paymentExpand"></param>
+        /// <returns></returns>
+        Task<ITrustlyPaymentResponse> Create(TrustlyPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);
+
+        ///// <summary>
+        /////     Gets an existing swish payment.
+        ///// </summary>
+        ///// <param name="id"></param>
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)